doc/source/whatsnew/v1.0.2.rst

```diff
@@ -53,4 +55,4 @@ Bug fixes
 Contributors
 ~~~~~~~~~~~~
 
-.. contributors:: v1.0.1..v1.0.2|HEAD
\ No newline at end of file
+.. contributors:: v1.0.1..v1.0.2|HEAD
```

Comment:
Is this modified here for a reason? If not can you revert?

Reply:
That was not on purpose, I will revert.

Reply:
@WillAyd done. Thank you for the feedback.